CemrgApp/Plugins/kcl.cemrgapp.mmcwplugin/src/internal/MmcwView.cpp

```diff
@@ -683,7 +683,7 @@ void MmcwView::Tracking() {
     }//_if
 
     //Ask for user input to set the parameters
-    QDialog* inputs = new QDialog(0, 0);
+    QDialog* inputs = new QDialog(0, Qt::WindowFlags());
```

Comment:
I have no strong feelings either way and would just leave this as-is. (It’s probably inconsistent in a bunch of other places across the codebase as well; so if we do want to make it consistent, let’s use something like [clang-format](https://clang.llvm.org/docs/ClangFormat.html) instead of manual edits.)